src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: 50%;
+  margin-right: 4px;
```

Comment:
The `margin-right` property should be the last one in the block since it does not relate to the background or size. It's a good practice to follow a consistent order of properties, such as positioning, box model, typography, visual, etc.